snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/SnomedConceptUpdateRequest.java

```diff
@@ -201,18 +207,52 @@ private boolean isDifferentToPreviousRelease(Concept concept, SnomedConcept rele
 	}
 
 	private boolean updateDefinitionStatus(final TransactionContext context, final Concept concept) {
-		if (null == definitionStatus) {
-			return false;
+		final Set<String> newOwlAxiomExpressions = Optional.ofNullable(members)
+				.map(Collection::stream)
+				.orElseGet(Stream::empty)
+					.filter(member -> SnomedRefSetType.OWL_AXIOM == member.type() || Concepts.REFSET_OWL_AXIOM.equals(member.getReferenceSetId()))
+					.map(member -> (String) member.getProperties().get(SnomedRf2Headers.FIELD_OWL_EXPRESSION))
+					.collect(Collectors.toSet());
+		
+		final String newDefinitionStatusId;
+		if (!newOwlAxiomExpressions.isEmpty()) {
+			// Calculate the definition status
+			newDefinitionStatusId = SnomedOWLAxiomHelper.getDefinitionStatusFromExpressions(newOwlAxiomExpressions).getConceptId();
+		} else {
+			final String incomingDefinitionStatusId = definitionStatus == null ? DefinitionStatus.PRIMITIVE.getConceptId() : definitionStatus.getConceptId();
```

Comment:
If the `definitionStatus` is `null`, just return from the `updateDefinitionStatus()` method. `Null` value indicates no update on the given property.